tasks/cephfs/vstart_runner.py

```diff
@@ -257,7 +257,7 @@ def _get_pid(self):
         Return PID as an integer or None if not found
         """
         ps_txt = self.controller.run(
-            args=["ps", "-xu"+str(os.getuid())]
+            args=["ps", "-xwu"+str(os.getuid())]
```

Comment:
Let's use ``ww`` instead of ``w`` (make it unlimited instead of exchanging one limit for another)